modules/ROOT/pages/install-add-tasks.adoc

```diff
@@ -32,17 +30,18 @@ You must forward logs to a 3rd-party log solution like Splunk. It's not supporte
 
 == Set up SMTP
 
-In order to handle e-mail alerts for both Runtime Manager and API Manager, you must have an SMTP server configured on your network. You must direct your local installation of Anypoint Platform to this server from the Access Management panel.
-
+To handle e-mail alerts for both Runtime Manager and API Manager, you must have an SMTP server configured on your network. You must direct your local installation of Anypoint Platform to this server from the Access Management panel. See xref:access-management-smtp.adoc[Configure SMTP].
```

Comment:
> See xref:access-management-smtp.adoc[Configure SMTP].

Our link checker is flagging this as a broken link. 
Based on the working link in the `nav.adoc` file,  and the actual name of the file, this line should be:

> See xref:access-management-SMTP.adoc[Configure SMTP].

**OR** We could rename the filename to use all letter case. You call @sleslie23